AnkiDroid/src/main/java/com/ichi2/libanki/sched/Sched.java

```diff
@@ -1608,19 +1608,24 @@ public int eta(int[] counts, boolean reload) {
 
 
     /**
-     * This is used when card is currently in the reviewer, to adapt the counts by removing this card from it.*/
+     * This is used when card is currently in the reviewer, to adapt
+     * the counts by removing this card from it.
+     *
+     * @param discardCard A card sent to reviewer that should not be
+     * counted.
```

Comment:
You have very strict line length limits set up I think. I think breaking these lines decreases readability and if the editor did that because of limits, you should bump the limits. 80 chars isn't a hard limit anywhere anymore I don't think, 100 or 120 when really needed is okay by me

```suggestion
     * Used when card is currently in the reviewer, adapts the counts by removing this card from it.
     *
     * @param discardCard A card sent to reviewer that should not be counted.
```

That can be taken as a general comment, I've noticed it before in your patches, but this one was the most glaring because I'm not sure javadoc `@param` annotations even work when broken like that, so your editor limit I think actually broke javadoc